modules/release-notes/pages/index.adoc

```diff
@@ -124,6 +135,47 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.3
+
+==== Crashes and Deadlocks
+* Fixed crash when query thread would keep running after `JobListener` is shutdown (CORE-1730).
+* Fixed GPE crash when using `INSERT` with an `UNKNOWN` accumulator (GLE-5038).
+* Fixed crash when running interpret query with dynamic value (GLE-5600).
+* Fixed JSON Viewer crash on browser when displaying a large response (TOOLS-2283).
+
+==== Functionality
+* Fixed issue of missing files in AdminPortal when downloading an output (TOOLS-2499).
+* Fixed issue primary id could not set a color config for a vertex (TOOLS-2478).
+* Fixed issue where administrator could privilege of vertex `primary_id` in Role Management page of Admin Portal (TOOLS-2430).
+* Fixed issue where error messages of abort requests created large file sizes (CORE-2705).
+* Fixed issue where GPE was still in warmup while running chaos testing (CORE-2670).
```

Comment:
This one feels odd because we cite the condition as "chaos testing" which is a type of testing, not a use case.  Does the ticket explain what was the root cause?

Reply:
changed with latest commit.